CHANGELOG.md

```diff
@@ -0,0 +1,182 @@
+# Changelog
```

Comment:
Please add a step to the "Going final" list if you haven't already.

Reply:
I will once this has been accepted.